src/components/Calculator.css

```diff
@@ -0,0 +1,55 @@
+.ContainerCalc {
+  width: 50%;
+  display: flex;
+  flex-direction: column;
+}
+
+.Result {
+  background: gray;
+  height: 40px;
+  margin: 0;
+  padding-right: 1%;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.Totalbuttons {
+  display: flex;
+}
+
+.ButtonsCont {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.Numbers {
+  width: 100%;
+  display: grid;
```

Comment:
- [ ] Please make sure your design matches the given design and fix the below issues.
  - Please remove the gap between the digit buttons and the operational buttons.
  - Please remove the colored border from the operational buttons and use the color `#F5913E` for the background.


### Your design

![image](https://user-images.githubusercontent.com/83803414/219111770-effb5cad-ac31-487b-8fe1-1a4f23b4745c.png)

### Given design

![image](https://user-images.githubusercontent.com/83803414/219111822-57386558-69d4-4d40-a49b-87a80605865a.png)